kf_lib_data_ingest/etl/extract/extract.py

```diff
@@ -172,11 +169,21 @@ def _run(self):
                 **(extract_config.loading_params)
             ).applymap(intsafe_str)
 
+            self.logger.debug(
+                "Loaded DataFrame with dimensions %s", df_in.shape
+            )
+
             # extraction
             df_out = self._chain_operations(
                 df_in, extract_config.operations
             )
 
+            # split value lists into separate rows
+            df_out = split_df_rows_on_splits(
+                                                df_out.reset_index()
+                                            ).set_index('index')
+            del df_out.index.name
```

Comment:
Why is this needed?

Reply:
Having a named index just offends me. No other reason.